tailf_test.go

```diff
@@ -195,88 +215,275 @@ func canFollowFileFromEnd(t *testing.T, filename string, file *os.File) error {
 	errc := make(chan error, 1)
 	go func() {
 		n, err := io.ReadAtLeast(follow, make([]byte, 1), 1)
-		t.Logf("read %d bytes after closing", n)
+		t.Logf("Read %d bytes after closing", n)
 		errc <- err
 	}()
 
 	if err := follow.Close(); err != nil {
-		t.Errorf("failed to close follower: %v", err)
+		t.Errorf("Failed to close tailf.follower: %v", err)
 	}
 
 	got := string(data)
 	if want != got {
-		t.Errorf("want %v, got %v", want, got)
+		t.Errorf("Wanted '%v', got '%v'", want, got)
 	}
 
 	err = <-errc
 	if err != io.EOF {
-		t.Errorf("expected EOF after closing follower, got %v", err)
+		t.Errorf("Expected EOF after closing the follower, got '%v' instead", err)
 	}
 
 	return nil
 }
 
-func TestFollowTruncation(t *testing.T) { withTempFile(t, canFollowTruncation) }
+func TestFollowTruncation(t *testing.T) { withTempFile(t, time.Millisecond*150, canFollowTruncation) }
 
 func canFollowTruncation(t *testing.T, filename string, file *os.File) error {
 	follow, err := tailf.Follow(filename, false)
 	if err != nil {
-		t.Fatalf("failed to create follower: %v", err)
+		t.Fatalf("Failed creating tailf.follower: %v", err)
```

Comment:
not a big fan of capitalization of all the messages, it's not a style that I favor and it's making the PR harder to read.